pilot/pkg/security/authz/builder/testdata/allow-none-out.yaml

```diff
@@ -0,0 +1,12 @@
+name: envoy.filters.http.rbac
+typedConfig:
+  '@type': type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC
+  rules:
+    policies:
```

Comment:
previously this is generated as 
```
rules: {}
```
which means it will never match, now we generate this in a more explicit format.